OMCompiler/Compiler/Template/MMToJulia.tpl

```diff
@@ -183,29 +188,45 @@ match classDef
         ':'
     '= enumeration(<%enum_str%>)'
   case PDER(__) then
-    let func_str = AbsynDumpTpl.dumpPath(functionPath)
+    let func_str = dumpPathJL(functionPath)
     '= der(<%func_str%>, <%derivedVariables ;separator=", "%>)'
   case OVERLOAD(__) then
-    '= overload(<%pathLst |> path => AbsynDumpTpl.dumpPath(path); separator=", "%>)'
+    '= overload(<%pathLst |> path => dumpPathJL(path); separator=", "%>)'
   else error(sourceInfo(), "SCodeDump.dumpClassDef: Unknown class definition.")
 end dumpClassDef;
 
 template dumpTypeSpec(Absyn.TypeSpec typeSpec)
 ::=
 match typeSpec
   case TPATH(__) then
-    let path_str = AbsynDumpTpl.dumpPath(path)
+    let path_str = dumpPathJL(path)
     let arraydim_str = AbsynDumpTpl.dumpArrayDimOpt(arrayDim)
     '<%path_str%><%arraydim_str%>'
   case TCOMPLEX(__) then
     let path_str = (match path
        case IDENT(name="list") then 'List'
-       else AbsynDumpTpl.dumpPath(path))
+       else dumpPathJL(path))
     let ty_str = (typeSpecs |> ty => dumpTypeSpec(ty) ;separator=", ")
     let arraydim_str = AbsynDumpTpl.dumpArrayDimOpt(arrayDim)
     '<%path_str%>{<%ty_str%>}<%arraydim_str%>'
 end dumpTypeSpec;
 
+template dumpTypeSpecAF(Absyn.TypeSpec typeSpec)
```

Comment:
Very ugly, best solution is a function calling a function, and that function takes two parameters?